test/generators/suspenders/accessibility_generator_test.rb

```diff
@@ -0,0 +1,89 @@
+require "test_helper"
+require "generators/suspenders/accessibility_generator"
+
+module Suspenders
+  module Generators
+    class AccessibilityGeneratorTest < Rails::Generators::TestCase
+      include Suspenders::TestHelpers
+
+      tests Suspenders::Generators::AccessibilityGenerator
+      destination Rails.root
+      setup :prepare_destination
+      teardown :restore_destination
+
+      test "generator runs without errors" do
+        assert_nothing_raised do
+          run_generator
+        end
+      end
```

Comment:
I'm always skeptical of the value of tests that use `assert_nothing_raised`. The absence of an error a fairly weak assertion to make outright, especially when there are other tests in the same file that make assertions about behavior and side effects that bake in this assertion, since they execute and pass without Exceptions.